tests/OK/inductive.lp

```diff
@@ -641,31 +641,6 @@ assert p pnil pcons a n x v
 ⊢ ind_Vec p pnil pcons _ _ (Vcons a n x v)
 ≡ pcons a n x v (ind_Vec p pnil pcons a n v);
 
-////////////////// Type Bush (nested inductive type)
```

Comment:
If I remember correctly, this was an important example for Amelie's work, shouldn't we at least keep some record of it?

Reply:
The problem is that the induction principle generated by this type definition doesn't seem to be usable. More investigations need to be done on this kind of type definitions. With the current code, this is not a reasonable example to keep at the moment. It will be easy to add it again later if needed.